test/typescript/compose.ts

```diff
@@ -36,4 +36,4 @@ const t11: number = compose(stringToNumber, numberToString, stringToNumber,
 
 
 const funcs = [stringToNumber, numberToString, stringToNumber];
-const t12 = compose(...funcs)('bar', 42, true);
```

Comment:
This appears to have just been a bug in the old test case, which the newer version of TypeScript caught.